cloudstack/python/tar/slipstream_cloudstack/CloudStackClientCloud.py

```diff
@@ -339,9 +350,8 @@ def _vm_get_ram(self, vm_instance):
 
     @override
     def _vm_get_root_disk(self, vm_instance):
-        size = self._get_vm_size(vm_instance)
-        if size:
-            return size.disk
+        volume = self._find_root_disk_volume(vm_instance.id)
+        return volume.size / (1024 * 1024 * 1024) if volume else None
```

Comment:
I think you might want to wrap into float or else you may loose precision in P2.
```
>>> 10000000001 / (1024 * 1024 * 1024)
9
>>> float(10000000001) / (1024 * 1024 * 1024)
9.313225747086108
>>>
```